core/jest.setup.js

```diff
@@ -2,3 +2,4 @@ require("@grouparoo/spec-helper/dist/lib/environment");
 
 // tell `@grouparoo/spec-helper` to load everything from `src` in core, so we can test without re-compiling
 process.env.GROUPAROO_SPEC_HELPER_LOAD_PATH = "src";
+process.env.TZ = "UTC";
```

Comment:
This theoretically should make CI consistent with my local but I'm not sure it's going to do it.